infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -2385,6 +2385,26 @@ class PublicationDao(
             )
         return jdbcTemplate.query(sql, params) { rs, _ -> rs.getIntId("location_track_id") }
     }
+
+    fun fetchPublishedTrackNumbersAfterMoment(
+        exclusiveStartMoment: Instant,
+        inclusiveEndMoment: Instant,
+    ): List<IntId<LayoutTrackNumber>> {
+        val sql =
+            """
+            select distinct ptn.id as track_number_id
+            from publication.track_number ptn
+              join publication.publication publication on ptn.publication_id = publication.id
```

Comment:
Tästä taisikin olla dailyssä jo puhe että pitäisi huomioida designit (vaikka vain `design_id is null` ehdolla)